dynamic.js

```diff
@@ -47,4 +47,133 @@ function closecContainer() {
+    badgesCard: ['html', 'bootstrap', 'Ruby'],
+    imageURL: './images/popup-img.png',
+    liveURL: 'https://www.github.com',
+    projectURL: 'https://www.github.com/ahmedadel56',
+  },
+  {
+    headerCard: 'Data Dashboard Healthcare',
+    textCard: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    badgesCard: ['html', 'bootstrap', 'Ruby'],
+    imageURL: './images/popup-img.png',
+    liveURL: 'https://www.github.com',
+    projectURL: 'https://www.github.com/ahmedadel56',
+  },
+  {
+    headerCard: 'Website Protfolio',
+    textCard: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    badgesCard: ['html', 'bootstrap', 'Ruby'],
+    imageURL: './images/popup-img.png',
+    liveURL: 'https://www.github.com',
+    projectURL: 'https://www.github.com/ahmedadel56',
+  },
+  {
+    headerCard: 'Profesional Art Printing Data More',
+    textCard: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    badgesCard: ['html', 'bootstrap', 'Ruby'],
+    imageURL: './images/popup-img.png',
+    liveURL: 'https://www.github.com',
+    projectURL: 'https://www.github.com/ahmedadel56',
+  },
+  {
+    headerCard: 'Data Dashboard Healthcare',
+    textCard: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    badgesCard: ['html', 'bootstrap', 'Ruby'],
+    imageURL: './images/popup-img.png',
+    liveURL: 'https://www.github.com',
+    projectURL: 'https://www.github.com/ahmedadel56',
+  },
+  {
+    headerCard: 'Website Protfolio',
+    textCard: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    badgesCard: ['html', 'bootstrap', 'Ruby'],
+    imageURL: './images/popup-img.png',
+    liveURL: 'https://www.github.com',
+    projectURL: 'https://www.github.com/ahmedadel56',
+  },
+  {
+    headerCard: 'Profesional Art Printing Data More',
+    textCard: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    badgesCard: ['html', 'bootstrap', 'Ruby'],
+    imageURL: './images/popup-img.png',
+    liveURL: 'https://www.github.com',
+    projectURL: 'https://www.github.com/ahmedadel56',
+  },
+];
+
+function loadCards(cardsData = mockData) {
+  cardsData.forEach((element) => {
+    const card = document.createElement('div');
+    card.className = 'popup';
+    const popupHead = document.createElement('div');
+    popupHead.className = 'popup-head';
+    const header = document.createElement('h2');
+    header.className = 'popup-header';
+    header.innerText = element.headerCard;
+    const closeCard = document.createElement('a');
+    closeCard.innerHTML = '&times;';
```

Comment:
- [ ] As the first reviewer requested, I think you need to adjust the size of the elements of the pop-up. When styling, always keep in mind that users may have different screen sizes.

![image](https://user-images.githubusercontent.com/43698511/132775475-1a3d3187-f7d0-4b45-8ed9-41b5eda0ded7.png)

Reply:
> * [x]  As the first reviewer requested, I think you need to adjust the size of the elements of the pop-up. When styling, always keep in mind that users may have different screen sizes.
> 
> ![image](https://user-images.githubusercontent.com/43698511/132775475-1a3d3187-f7d0-4b45-8ed9-41b5eda0ded7.png)

@safafa okay check them again please

Reply:
I fixed the issue so please check it again

Reply:
@ahmedadel56 I checked it everything is inside the pop-up but there are still some issues I think you should solve 
![image](https://user-images.githubusercontent.com/43698511/132836275-304eceaa-830f-450d-8609-1db7232b1967.png)

- I suggest you to adjust the distance between the title and the tags, refer to the original design.

- The scroll bars are showing because there's an overflow. I think you should fix it

![image](https://user-images.githubusercontent.com/43698511/132836409-53907594-ebe9-4f5c-96d7-184a67be7c5b.png)

Reply:
> @ahmedadel56 I checked it everything is inside the pop-up but there are still some issues I think you should solve
> ![image](https://user-images.githubusercontent.com/43698511/132836275-304eceaa-830f-450d-8609-1db7232b1967.png)
> 
> * I suggest you to adjust the distance between the title and the tags, refer to the original design.
> * The scroll bars are showing because there's an overflow. I think you should fix it
> 
> ![image](https://user-images.githubusercontent.com/43698511/132836409-53907594-ebe9-4f5c-96d7-184a67be7c5b.png)

@Safafa I have fixed the issue so please check them again